### PR TITLE
Init Symfony 4 skeleton with first Docker env

### PR DESCRIPTION
**Pull request by cvilleger**
Related to #1 

- Symfony 4 (Skeleton)
- Docker dev env (Nginx + PHP 7.2)

### CODE REVIEW

File: docker-compose.override.yml.dist [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: "3.5"
```
Comment (tdutrion) [inline docker-compose.override.yml.dist]: Why 3.5?

Comment (cvilleger) [reply docker-compose.override.yml.dist]: Yeah... I'm forcing people to have a recent version of Docker Compose (at least 17.12.0+ from the past year). Does that make me a bad guy ? :innocent:

Comment (cgrandval) [reply docker-compose.override.yml.dist]: I think the question was more, why not 3.7

Comment (tdutrion) [reply docker-compose.override.yml.dist]: Then I would probably go for the lowest number you can find with all the features you are using, but that's going to be too much work...

**Pull request merged**